### PR TITLE
doc/start: update documenting-ceph.rst (1 of x)

### PR DESCRIPTION
**Pull request by zdover23**
This PR updates the content on documenting-ceph,
which is, as of December 2021, in need of an
update.

This is the first of what I estimate to be three
to five PRs against this .rst file.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/start/documenting-ceph.rst [hunk 1]
```diff
@@ -2,26 +2,50 @@
...
-instructions will help the Ceph project immensely.
-
-The Ceph documentation source resides in the ``ceph/doc`` directory of the Ceph
-repository, and Python Sphinx renders the source into HTML and manpages. The
-https://docs.ceph.com link currently displays the ``master`` branch by default,
-but you may view documentation for older branches (e.g., ``mimic``) by substituting
-``latest`` in the URL with the branch name you prefer.
-
+You can help the Ceph project by contributing to the documentation.  Even
+small contributions help the Ceph project: contributions such as fixing
```
Comment (ljflores) [inline doc/start/documenting-ceph.rst:6]:
```suggestion
small contributions help the Ceph project, such as fixing
```

File: doc/start/documenting-ceph.rst [hunk 2]
```diff
@@ -2,26 +2,50 @@
...
+The easiest way to suggest a correction to the documentation is to send an
+email to `ceph-users@ceph.io`. Include the string "ATTN: DOCS" or
+"Attention: Docs" or "Attention: Documentation" in the subject line.  In
+the body of the email, include the text to be corrected (so that I can find
+it in the repo) and include your correction.
+
+If you prefer making a pull request to sending an email, you can make a
+pull request to improve the documentation. The instructions for making a
+pull request against the Ceph documentation are available in the section
+:ref:`making_contributions`.
```
Comment (ljflores) [inline doc/start/documenting-ceph.rst:18]:
```suggestion
Another way to suggest a documentation correction is to make a pull request.
The instructions for making a pull request against the Ceph documentation are 
available in the section :ref:`making_contributions`.
```

**Review by ljflores — APPROVED** [review]
Left a few suggestions to shorten some of the sentences, but other than that, looks good to me! And obviously you're the documentation expert, so take my suggestions with a grain of salt.

**Comment by ljflores** [comment]
Also, the "make check (arm64)" error is known and has been reported: https://tracker.ceph.com/issues/53474

**Comment by ljflores** [comment]
jenkins retest make check arm64

**Comment by ljflores** [comment]
jenkins test make check arm64

File: doc/start/documenting-ceph.rst (hunk 2 above)
Comment (zdover23) [reply doc/start/documenting-ceph.rst:18]: Yeah, this is better.

File: doc/start/documenting-ceph.rst (hunk 1 above)
Comment (zdover23) [reply doc/start/documenting-ceph.rst:6]: Good point.

**Comment by ljflores** [comment]
The new make check arm64 failure is yet another known bug: https://tracker.ceph.com/issues/53451
Ernesto has a PR open to temporarily disable the test.

**Pull request merged**